include/dol/bases/d_base.hpp

```diff
@@ -1,49 +1,44 @@
 #pragma once
-
 #include <dol/framework/f_base.hpp>
 #include <dol/cLib/c_owner_set.hpp>
-
+/// @file
 
 /// @brief An extension of fBase_c with base kind and name strings.
 /// @details All bases should inherit from this class, not from fBase_c.
 class dBase_c : public fBase_c, public cOwnerSetMg_c {
     public:
-        /// @copydoc fBase_c::fBase_c
         dBase_c();
-        
-        virtual int preCreate(); ///< @copydoc fBase_c::preCreate
-        virtual void postCreate(fBase_c::MAIN_STATE_e status); ///< @copydoc fBase_c::postCreate
 
-        virtual int preDelete(); ///< @copydoc fBase_c::preDelete
-        virtual void postDelete(fBase_c::MAIN_STATE_e status); ///< @copydoc fBase_c::postDelete
+        virtual int preCreate();
+        virtual void postCreate(fBase_c::MAIN_STATE_e status);
+
+        virtual int preDelete();
+        virtual void postDelete(fBase_c::MAIN_STATE_e status);
 
-        virtual int preExecute(); ///< @copydoc fBase_c::preExecute
-        virtual void postExecute(fBase_c::MAIN_STATE_e status); ///< @copydoc fBase_c::postExecute
+        virtual int preExecute();
+        virtual void postExecute(fBase_c::MAIN_STATE_e status);
 
-        virtual int preDraw(); ///< @copydoc fBase_c::preDraw
-        virtual void postDraw(fBase_c::MAIN_STATE_e status); ///< @copydoc fBase_c::postDraw
+        virtual int preDraw();
+        virtual void postDraw(fBase_c::MAIN_STATE_e status);
 
-        virtual ~dBase_c(); ///< @copydoc fBase_c::~fBase_c
+        virtual ~dBase_c();
         
         /// @brief Gets a string describing the kind of this base.
         virtual const char *getKindString() const;
 
-        /// @copydoc fManager_c::searchBaseByProfName
```

Comment:
I think this is still needed because `fManager_c` is not a subclass of `dBase_c`.